acme/acme/client.py

```diff
@@ -509,15 +550,19 @@ class ClientNetwork(object):  # pylint: disable=too-many-instance-attributes
     JSON_ERROR_CONTENT_TYPE = 'application/problem+json'
     REPLAY_NONCE_HEADER = 'Replay-Nonce'
 
-    def __init__(self, key, alg=jose.RS256, verify_ssl=True,
-                 user_agent='acme-python', timeout=DEFAULT_NETWORK_TIMEOUT):
+    def __init__(self, key, account=None, alg=jose.RS256, verify_ssl=True,
+                 user_agent='acme-python', timeout=DEFAULT_NETWORK_TIMEOUT,
+                 acme_version=1):
```

Comment:
I think it will be pretty easy to mess this up by forgetting to set `acme_version` here when using `Client2`. Maybe we could put some sort of assertion over in `Client2` that `acme_version` is 2, or something.

Reply:
update: we probably want to remove acme_version=1 and self.acme_version = acme_version

Reply:
(still need to do this)

Reply:
Can you remind me in more detail what we settled on for "this"?

Reply:
Remove the now-dead code of `acme_version=1` and `self.acme_version = acme_version`

Because the `ClientNetwork` object doesn't need to know about acme version anymore, since it's all detailed in `post`